### PR TITLE
feat: yaml linting

### PR DESCRIPTION
**Pull request by eugene-davis**
Apply linting to the Pipeline yaml files.

### CODE REVIEW

**Comment by eugene-davis** [comment]
See build https://dev.azure.com/tomtomweb/GitHub-TomTom-International/_build/results?buildId=3204 for a passing test run.

**Review by ghost — DISMISSED** [review]

File: jobs/python.setup.py.yml [hunk 1]
```diff
@@ -18,76 +19,76 @@ parameters:
...
-        python.version: '3.5'
-      Python36:
-        python.version: '3.6'
-      Python37:
-        python.version: '3.7'
-  pool:
-    vmImage: 'ubuntu-16.04'
-  steps:
-  - template: ../steps/python/python.build.yml
+  - job: 'Linux'
```
Comment (martinm82) [inline jobs/python.setup.py.yml:21]: Why these formatting changes?

Comment (eugene-davis) [reply jobs/python.setup.py.yml:21]: It was not actually correctly formatted yaml

Comment (martinm82) [reply jobs/python.setup.py.yml:21]: It was not wrong, just different but I agree, let's keep it consistent and good to have a tool doing it for us 👍

File: steps/python/python.build.yml [hunk 2]
```diff
@@ -2,46 +2,46 @@
 #
 # Build, lint and test Python modules based on setuptools (setup.py)
 #
-
+---
```
Comment (martinm82) [inline steps/python/python.build.yml:5]: Sometimes we have a newline after that, sometimes not. The formatter is not checking that?

**Comment by martinm82** [comment]
Very nice to have the linter in place 👍

File: steps/python/python.build.yml (hunk 2 above)
Comment (eugene-davis) [reply steps/python/python.build.yml:5]: No, unfortunately it is not. I will try to make them consistent manually.

Comment (eugene-davis) [reply steps/python/python.build.yml:5]: Fixed. We might look into whether we can add a rule for this in the future.

**Comment by eugene-davis** [comment]
> Very nice to have the linter in place +1

FYI, if we go the route of putting comments onto PRs, we should be able to take the output from this linter pretty easily

**Review by martinm82 — APPROVED** [review]

**Comment by martinm82** [comment]
Awesome change 👍

**Pull request merged**